app/src/main/java/com/onboarding/wheatherapp/services/api/WeatherApi.kt

```diff
@@ -0,0 +1,16 @@
+package com.onboarding.wheatherapp.services.api
+
+import com.onboarding.wheatherapp.services.response.WeatherResponse
+import retrofit2.Call
+import retrofit2.http.GET
+import retrofit2.http.Query
+
+interface WeatherApi {
+
+    @GET("data/2.5/forecast")
```

Comment:
you can put data/2.5/ in the base URL too. Its the same for all the end points

Reply:
Solved!